test/test.py

```diff
@@ -277,25 +258,25 @@ def test_fetch_parallel(self):
                     previous_line = line
             self.assertTrue(finish_precedes_acquire)
 
+def webserver_for_test_send(queue):
```

Comment:
Same comment as for `webserver_for_test_fetch`: why was this moved?

Reply:
The same as above. I encountered an error on running the test:
~~~
AttributeError: Can't pickle local object 'TestSend.setUp.<locals>.webserver'
~~~